app/controllers/holds_controller.rb

```diff
@@ -167,14 +167,21 @@ def past_date
     end
 
```

Comment:
I think this could use a comment because it's kinda weird.

```suggestion
    # Fetch a hold lookup as many times as necessary up to 5 seconds to avoid `nil` item in response. 
```